### ISSUE TITLE
[Sample App] Add aria regions

### ISSUE THREAD

**Issue by dpayk** [open-pioneer/trails-openlayers-base-packages#186]
For better accessibility there should be regions in the sample to allow a faster navigation while using screenreaders as described here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role

![Image](https://github.com/open-pioneer/trails-openlayers-base-packages/assets/13886355/836f7e9f-b119-401a-9f13-dac6c28f0a8b)
